scripts/setup-dependencies

```diff
@@ -2,7 +2,7 @@
 set -ex
 cd "$(dirname "$0")/.."
 
-pip install -r core/requirements.txt
+pip install -r core/requirements.txt --root-user-action
```

Comment:
```suggestion
pip install -r core/requirements.txt --root-user-action=ignore
```

This needs to be added to all of them. Seems like this option saw changes in different pip versions.